content/blog/2022-01-14-users-guide-to-contribution-ross/index.md

```diff
@@ -0,0 +1,45 @@
+
+### Technical Development
+
+A large scale, complicated, open source software engineering project and [source code](https://github.com/galaxyproject) forms the core of the project deliverables.  Like the project itself, the code repository is complicated and made up of many interacting components and dependencies. The large community that has become engaged with that code now helps sustain the project, in part by providing a pool of talented external contributors, large enough to place Galaxy among the top 2% of OpenHub projects for [developer activity](https://www.openhub.net/p/galaxybx/factoids#FactoidAgeVeryOld)
+
+If you would like to learn more about how you can contribute technical expertise, The [Working Groups](https://galaxyproject.org/community/wg/) page is the best place to start as there are a wide variety of specialised areas to choose from. Contact the specific Working Group convenor to learn more.
+
+### Community Development
+
+Like the source code, the community provides opportunities for contributions, such as collaborations to improve the sustainability, usefulness and growth of the community itself. In many cases, these projects will also involve technical aspects, but they rely primarily on other specialised scientific and organisational skills and experience.
+
+The [Outreach and Training WG](https://galaxyproject.org/community/wg/) addresses many issues that are community centered. This WG may be of interest to potential contributors to educational and training activities because it includes [Galaxy Training Network](https://training.galaxyproject.org/) collaborations
+
+The Global Galaxy Steering Committee
+[(GGSC)](https://galaxyproject.org/community/steering/) is a formal Galaxy governance structure concerned with community activities. It consists of investigators and other Galaxy community contributors, who plan and manage projects and initiatives focussed on supporting and extending the community. The GGSC has now been in existence for a little more than a year, and offers a platform for new community initiatives. Community working groups are active in a broad range of topics such as mentoring, onboarding new investigators, and sustaining specific sub-communities such as small server administrators. New ideas are always welcome.
+
+The GGSC Working Groups are open to community contributors. If you would like to learn more about any of them, contact the relevant Community [Working Group](https://galaxyproject.org/community/steering/) convenor. In addition, the GGSC seeks community leaders as new members. The main requirements are vision, and willingness to invest time and effort to see that vision realised. If you would like to learn more, please email the [GGSC chair](mailto:ross.lazarus@gmail.com)
```

Comment:
```suggestion
The GGSC Working Groups are open to community contributors. If you would like to learn more about any of them, contact the relevant Community [Working Group](https://galaxyproject.org/community/steering/#subcommittees) convenor. In addition, the GGSC seeks community leaders as new members. The main requirements are vision, and willingness to invest time and effort to see that vision realised. If you would like to learn more, please email the [GGSC chair](mailto:ross.lazarus@gmail.com).
```

Reply:
We called them "subcommittees" in the GGSC page. Working group could be confusing I think

Reply:
I might change that too then!